.mvn/develocity.xml

```diff
@@ -0,0 +1,30 @@
+<develocity
+		xmlns="https://www.gradle.com/develocity-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="https://www.gradle.com/develocity-maven https://www.gradle.com/schema/develocity-maven.xsd">
+	<server>
+		<url>https://ge.spring.io</url>
+	</server>
+	<buildScan>
+		<backgroundBuildScanUpload>false</backgroundBuildScanUpload>
```

Comment:
Is this something we want to run on a schedule or every time?
Have we tested how much time this will add to our CI builds?